docs/source/specs/openapi.json

```diff
@@ -1186,6 +1186,20 @@
               "format": "uuid"
             }
           },
+          {
+            "name": "admin_only",
+            "in": "query",
+            "description": "Get only admin users within an account.",
+            "required": false,
+            "schema": {
+              "type": "string",
```

Comment:
We're about to use this in Notifications. Is there a reason why this field is not a `boolean`?

Reply:
hi @gwenneg I believe it is boolean [now](https://github.com/RedHatInsights/insights-rbac/blob/ca5763f67a768aa5d88f73661004820f578dcc72/docs/source/specs/openapi.json#L1211)

Reply:
Nice. I should have checked the latest code 😅 Thanks for your help!

Reply:
yw, let me know if anything ❤️